### ISSUE TITLE
make the locust ds class  example shorter.

### ISSUE THREAD

**Issue by rfl-urbaniak** [BasisResearch/collab-creatures#61]
Re: convergence, makes sense -- I think it likely will make sense to make the example be a shorter duration, for the reasons you mention, but this doesn't need to block us posting the repository. Can you make an issue to investigate this in the future?

_Originally posted by @emackev in https://github.com/BasisResearch/collaborative-intelligence/pull/58#discussion_r1510344234_

**Comment by emackev** [comment]
I wanted to check with you before doing https://github.com/BasisResearch/collab-creatures/issues/52, so we don't get merge conflicts -- LMK if you're actively working on this and/or if it's ok for me to go ahead and start adding documentation.

**Comment by rfl-urbaniak** [comment]
Yes, I am actively working on this one, but if you restrict yourself to the `locust-approximate-pipeline` notebook for now, you're not really touching the DS stuff, so there should be no conflicts, which is what I recommend. Should have a PR addressing #61 tomorrow.

**Comment by emackev** [comment]
Sounds good! https://github.com/BasisResearch/collab-creatures/pull/68 is just the locust-approximate-pipeline notebook. I can make another PR later for documentation for the DS notebooks.
